kornia/filters/gaussian.py

```diff
@@ -59,10 +59,10 @@ def gaussian_blur2d(
         sigma = sigma.to(device=input.device, dtype=input.dtype)
 
     if separable:
-        ks = _unpack_2d_ks(kernel_size)
+        ky, kx = _unpack_2d_ks(kernel_size)
         bs = sigma.shape[0]
-        kernel_x = get_gaussian_kernel1d(ks[1], sigma[:, 1].view(bs, 1))
-        kernel_y = get_gaussian_kernel1d(ks[0], sigma[:, 0].view(bs, 1))
+        kernel_x = get_gaussian_kernel1d(kx, sigma[:, 1].view(bs, 1))
```

Comment:
should sigma follow the same convention ?

Reply:
I only rename the variables here for clarity. There is no change in the code logic.
ks[1] -> kx
ks[0] -> ky

The old behavior interpret sigma as y first then x. There is no change in convention.

Reply:
great